crates/aiken-lang/src/parser/lexer.rs

```diff
@@ -87,7 +87,8 @@ pub fn lexer() -> impl Parser<char, Vec<(Token, Span)>, Error = ParseError> {
         "trace" => Token::Trace,
         "error" => Token::ErrorTerm,
         "as" => Token::As,
-        "assert" => Token::Assert,
+        "assert" => Token::Expect,
```

Comment:
Backward-compatibility? You're being too kind to @alessandrokonrad.

Reply:
Replacing `assert` with `expect` is not a big deal actually. I could handle it ;)